### ISSUE TITLE
Select device button box is gone

### ISSUE THREAD

**Issue by mofada** [flutter/flutter-intellij#3105]
![tim 20190121094151](https://user-images.githubusercontent.com/22286246/51448319-cf177e00-1d60-11e9-924b-46c69e05bda9.jpg)
As you can see, my selection device button is gone.
I opened the android emulator first, then opened the ios emulator again. Then the select device button disappeared.

**Comment by jwren** [comment]
Please paste the output of running `flutter doctor -v` here (available from the command
line or from `Tools > Flutter > Flutter Doctor`). It will provide the version of the
Flutter framework as well as of the IntelliJ plugin.

**Comment by mofada** [comment]
@jwren  Thinks , `flutter doctor -v` Run as follows :

jiang:~ dc002$ flutter doctor -v
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.14.2 18C54, locale
    zh-Hans-CN)
    • Flutter version 1.0.0 at /Users/dc002/Software/flutter
    • Framework revision 5391447fae (8 weeks ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/dc002/Software/android-sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/dc002/Software/android-sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[!] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS platform side's plugin code that
        responds to your plugin usage on the Dart side.
        Without resolving iOS dependencies with CocoaPods, plugins will not work
        on iOS.
        For more info, see https://flutter.io/platform-plugins
      To install:
        brew install cocoapods
        pod setup

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.3.3
    • Dart plugin version 182.5124
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1248-b01)

[!] Connected device
    ! No devices available


-----------------------------------------------------------------
I open an android emulator alone or open an iphone emulator separately. But I open the android emulator first, then open the iphone emulator. The device selection box is gone.

**Comment by long1eu** [comment]
Hello! :D Anyone here?

**Comment by long1eu** [comment]
I solved this issue by making sure that the dart SDK is pointing to the Flutter installation and not the plain Dart one.

**Comment by mofada** [comment]
Oh, hello, I don’t seem to have encountered this problem recently.

**Comment by traitsai** [comment]
> I solved this issue by making sure that the dart SDK is pointing to the Flutter installation and not the plain Dart one.

I'm glad I read your comment otherwise this would have taken forever to figure out! Yes, the main.dart file was complaining that the dart SDK was not configured and gave the option to Download Dart SDK or Open Dart Settings. I tried both and eventually was able to download Dart SDK and put the path in the Dart settings. But after doing that the iOS and Android Simulator drop down menu disappeared! The Select Device drop down menu wasn't there anymore and the "Run" button was greyed out. The solution was to update the Flutter SDK Path (Android Studio --> Preferences --> Languages --> Flutter) was pointing to the Flutter installation folder. Once I applied that and removed the Dart SDK path it auto populated the Dart SDK path to the Flutter install Dart folder.
